### PR TITLE
feat(react-utilities): isHTMLElement to support all HTMLElement classes

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. Improves `isHTMLElement` helper method to support all `HTMLElement` classes
2. Stops using `instanceof` in favor of `isHTMLElement` on:
   - `react-radio`
   - `react-menu`
   - `react-positioning` 

This is required as simply using `instanceof` might be problematic while operating with [multiple realms](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms)

Fixes https://github.com/microsoft/fluentui/issues/27159

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Avatar-mount_base.html">1330</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Avatar-mount.html">1328</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Button-mount_base.html">942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Button-mount.html">952</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Field-mount_base.html">2005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Field-mount.html">2013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProvider-mount_base.html">1648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProvider-mount.html">1666</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProviderWithTheme-mount_base.html">631</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProviderWithTheme-mount.html">636</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">591</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">601</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">633</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">630</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/InfoButton-mount_base.html">580</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/InfoButton-mount.html">564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/MakeStyles-mount_base.html">1888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/MakeStyles-mount.html">1915</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Persona-mount_base.html">3094</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/Persona-mount.html">3029</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/SpinButton-mount_base.html">2458</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27161/perf-test-react-components/SpinButton-mount.html">2440</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                 Change |
| :-------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr> | `148.051 kB`<br />`39.995 kB` | `148.116 kB`<br />`40.021 kB` | `65 B` <br />`26 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                                                                    |          `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr>                                                    |          `554 B`<br />`348 B` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `88.789 kB`<br />`26.532 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `90.467 kB`<br />`22.106 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `54.149 kB`<br />`14.571 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |   `18.064 kB`<br />`6.664 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `70.425 kB`<br />`19.075 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `25.746 kB`<br />`7.568 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `26.725 kB`<br />`7.874 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `28.318 kB`<br />`7.779 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `39.613 kB`<br />`9.935 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `46.795 kB`<br />`11.405 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `44.296 kB`<br />`11.255 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `52.846 kB`<br />`12.866 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `57.982 kB`<br />`11.853 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `83.548 kB`<br />`23.894 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `78.317 kB`<br />`22.433 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `9.045 kB`<br />`3.806 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `10.984 kB`<br />`4.509 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `9.852 kB`<br />`4.159 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/Checkbox.fixture.js'>Checkbox</abbr>                                                                              |   `35.895 kB`<br />`10.82 kB` |
| <samp>react-checkbox</samp> <br /> <abbr title='bundle-size/CheckboxField.fixture.js'>CheckboxField</abbr>                                                                    |    `42.8 kB`<br />`12.714 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `88.403 kB`<br />`28.031 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/ComboboxField.fixture.js'>ComboboxField</abbr>                                                                    |  `84.769 kB`<br />`27.762 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |   `87.09 kB`<br />`27.772 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `67.03 kB`<br />`18.099 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `206.019 kB`<br />`57.065 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `37.861 kB`<br />`12.034 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |   `92.72 kB`<br />`27.216 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |    `19.754 kB`<br />`6.68 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                                                                       |   `20.397 kB`<br />`7.053 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `11.259 kB`<br />`4.473 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `130.147 kB`<br />`39.329 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `25.664 kB`<br />`7.695 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/InputField.fixture.js'>InputField</abbr>                                                                             |  `35.613 kB`<br />`10.592 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `12.461 kB`<br />`4.57 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `12.306 kB`<br />`5.073 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `130.073 kB`<br />`39.444 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `133.249 kB`<br />`39.987 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `11.214 kB`<br />`4.271 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |   `61.21 kB`<br />`16.521 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `117.219 kB`<br />`35.735 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |   `11.377 kB`<br />`4.179 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.324 kB`<br />`2.129 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |    `23.791 kB`<br />`8.73 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                                        |   `15.965 kB`<br />`5.701 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressField.fixture.js'>ProgressField</abbr>                                                                    |   `26.415 kB`<br />`8.872 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `19.928 kB`<br />`6.832 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `35.392 kB`<br />`11.117 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |   `17.533 kB`<br />`6.426 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroupField.fixture.js'>RadioGroupField</abbr>                                                                   |   `27.762 kB`<br />`9.585 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |    `26.87 kB`<br />`8.761 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/SelectField.fixture.js'>SelectField</abbr>                                                                          |   `36.179 kB`<br />`11.28 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `35.816 kB`<br />`11.043 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/SliderField.fixture.js'>SliderField</abbr>                                                                          |  `45.717 kB`<br />`13.975 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `35.559 kB`<br />`10.351 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButtonField.fixture.js'>SpinButtonField</abbr>                                                              |  `44.565 kB`<br />`12.765 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |    `23.336 kB`<br />`7.18 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |    `30.943 kB`<br />`9.25 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/SwitchField.fixture.js'>SwitchField</abbr>                                                                          |  `37.805 kB`<br />`11.148 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `46.798 kB`<br />`12.557 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `137.652 kB`<br />`35.141 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `85.479 kB`<br />`21.159 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |    `84.8 kB`<br />`20.968 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `12.261 kB`<br />`4.819 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |    `15.571 kB`<br />`5.23 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `28.981 kB`<br />`9.071 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/TextareaField.fixture.js'>TextareaField</abbr>                                                                    |  `39.334 kB`<br />`12.008 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `49.054 kB`<br />`16.719 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                                       |          `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/05e84f28a54bb895074bd5aba07c4114b7be62bb'>05e84f28a54bb895074bd5aba07c4114b7be62bb</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-utilities/src/utils/isHTMLElement.ts [hunk 1]
```diff
@@ -7,11 +7,76 @@
  * might be problematic while operating with [multiple realms](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms)
  *
  */
-export function isHTMLElement(element?: unknown): element is HTMLElement {
+export function isHTMLElement<ConstructorName extends HTMLElementConstructorName = 'HTMLElement'>(
```
Comment (spmonahan) [inline packages/react-components/react-utilities/src/utils/isHTMLElement.ts:10]: It would be nice to add some tests for this function since it currently has none.

Comment (bsunderhus) [reply packages/react-components/react-utilities/src/utils/isHTMLElement.ts:10]: That's kind of hard though. I'm not sure how to simulate different realms on jest test environment.

Comment (spmonahan) [reply packages/react-components/react-utilities/src/utils/isHTMLElement.ts:10]: I think what you added is good. Thanks!

File: packages/react-components/react-utilities/src/utils/isHTMLElement.ts [hunk 2]
```diff
@@ -7,11 +7,76 @@
...
+  element?: unknown,
+  options?: { constructorName?: ConstructorName },
+): element is InstanceType<(typeof globalThis)[ConstructorName]> {
   const typedElement = element as Node | null | undefined;
   return Boolean(
     typedElement !== null &&
```
Comment (spmonahan) [inline packages/react-components/react-utilities/src/utils/isHTMLElement.ts:16]: Since you're using optional chaining on the next line I don't think you need this check.

File: change/@fluentui-react-menu-f76e8e18-42bd-4e30-92ec-977d7132c7aa.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-menu-f76e8e18-42bd-4e30-92ec-977d7132c7aa.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 8 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27161&commitId=a6a934481e67db3f651d3eee758f0b1a68891e38&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1884 screenshots unchanged, 0 screenshots with different dimensions and 8 screenshots with visible difference.


  <details><summary><b>unknown</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|1831483| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|1832333| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|1827930| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|1827931| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|1828108| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|1828114| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|1827814| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|1821426| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Review by khmakoto — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]
@bsunderhus please create a work item to create a lint rule to ban `instanceof` in our codebase, it more or less always breaks for multi-window
